providers/manage.rb

```diff
@@ -118,11 +118,23 @@ def chef_solo_search_installed?
         end
 
         if u['ssh_keys']
+          #on suse, i had errors if group didn't exist with the ssh_keys template, so try creating first
+          case node['platform_family']
+          when 'suse'
+            group new_resource.group_name do
+              gid new_resource.group_id
+              action :create
+            end
+          end
           template "#{home_dir}/.ssh/authorized_keys" do
             source "authorized_keys.erb"
             cookbook new_resource.cookbook
             owner u['username']
-            group u['gid'] || u['username']
+            if node['platform_family'] == 'suse' 
+              group new_resource.group_name
+            else 
+              group u['gid'] || u['username']
+            end
```

Comment:
It shows the intent, which is to have a `group` property, which is conditionally set.